invokeai/backend/lora.py

```diff
@@ -443,9 +445,12 @@ def _convert_sdxl_keys_to_diffusers_format(cls, state_dict: Dict[str, torch.Tens
                     new_state_dict[full_key] = value
                     not_converted_count += 1
             elif full_key.startswith("lora_te1_") or full_key.startswith("lora_te2_"):
-                # The CLIP text encoders have the same keys in both Stability AI and diffusers formats.
+                # The CLIP text encoders have the same keys in both Stability AI and diffusers formats
                 new_state_dict[full_key] = value
                 continue
+            elif full_key.startswith("diffusion_model_"):
+                new_key = full_key.replace("diffusion_model_", "")
+                new_state_dict[new_key] = value
```

Comment:
I don't think this is doing what you want. By simply removing the prefix, these keys will silently be skipped here: https://github.com/invoke-ai/InvokeAI/blob/caa7c0f2bd3748980e251f7c506c3d3592e47d3d/invokeai/backend/model_patcher.py#L117-L118

If the goal is to skip these keys, then we should track how many of them got skipped and log a warning.

Reply:
This is the only model I've found to have this key so far. You're right. It's probably just better to skip them given its just two layers. I'll close this PR for the time being an reopen another if I come across more LoRA's of this type.